core/src/main/java/org/apache/carbondata/core/statusmanager/SegmentUpdateStatusManager.java

```diff
@@ -455,6 +455,51 @@ public boolean accept(CarbonFile pathName) {
     return null;
   }
 
+  public Map<String, List<CarbonFile>> getDeleteDeltaFilesForSegment(final Segment seg) {
```

Comment:
remove getDeleteDeltaFilesList function

Reply:
Done.
Keep function name getDeleteDeltaFilesList and change the return type to Map<String, List<CarbonFile>>